tests/pipescript/pipescriptvar

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+./write_to_pipe.nexe $1 | ./read_from_pipe.nexe
```

Comment:
This and native could've joined if we just compiled them as write_to_pipe and read_from_pipe, but saved in different places; just like how we handle test suite.